index.html

```diff
@@ -785,6 +788,51 @@
           "site preferences".
         </p>
       </section>
+      <section>
+        <h3>
+          “Nice to have” images
+        </h3>
+        <p>
+          Images are often content, but there are instances when an image is an
+          enhancement to the document, but not a necessary part of the core
+          content (a.k.a. “nice to have“). As such, Authors may choose to load
+          an image only above a certain breakpoint in order to
+          optimize the readbility of the core content at certain sizes by
+          reducing the visual clutter. This is a similar use case to <a href="#art-direction">Art direction</a>, but distinct enough to warrant separation.
```

Comment:
It's not about lazy-loading, but about breakpoints. (the name "nice to have" might be confusing).

The use-case is about having some images "disappear" at certain breakpoints without introducing an extra overhead on the network (like a simple "display: none" would do). Basically, what Aaron wants is a way to define that for a certain breakpoint the image will not load at all. That is a form of art-direction.